src/views/studentregistration/studentregistration.jsx

```diff
@@ -132,6 +132,7 @@ class StudentRegistration extends React.Component {
                             onNextStep={this.handleAdvanceStep}
                         />
                         <Steps.DemographicsStep
+                            countryName={this.state.classroom && this.state.classroom.educator.profile.country}
```

Comment:
Ideally this would be:

```suggestion
                            countryName={this.state.classroom && this.state.classroom.educator && this.state.classroom.educator.profile && this.state.classroom.educator.profile.country}
```